OTP 23 has removed -lerl_interface

Hi,

OTP 23 has removed -lerl_interface from erl_interface-4.0, only -lei remains.

As a result, builds fail like this:

```
210 ===> Compiling prometheus_process_collector
211 make: Entering directory '/builds/deps/prometheus_process_collector/c_src'
212 g++ -O3 -finline-functions -fPIC -I /usr/lib/erlang/erts-11.0/include/ -I /usr/lib/erlang/lib/erl_interface-4.0/include -std=c++11 -Wall  -c -o prometheus_process_collector_nif.o prometheus_process_collector_nif.cc
213 g++ -O3 -finline-functions -fPIC -I /usr/lib/erlang/erts-11.0/include/ -I /usr/lib/erlang/lib/erl_interface-4.0/include -std=c++11 -Wall  -c -o prometheus_process_info_linux.o prometheus_process_info_linux.cc
214 cc prometheus_process_collector_nif.o prometheus_process_info_linux.o -shared -L /usr/lib/erlang/lib/erl_interface-4.0/lib -lerl_interface -lei -lstdc++ -o /builds/deps/prometheus_process_collector/c_src/../priv/prometheus_process_collector.so
215 /usr/bin/ld: cannot find -lerl_interface
216 collect2: error: ld returned 1 exit status
```

Thanks for maintaining this library!

--- a/c_src/Makefile
+++ b/c_src/Makefile
@@ -39,7 +39,7 @@ endif
 SOURCES = prometheus_process_collector_nif.cc $(PSOURCES)
 
 CXXFLAGS += -fPIC -I $(ERTS_INCLUDE_DIR) -I $(ERL_INTERFACE_INCLUDE_DIR) -std=c++11 -Wall
-LDLIBS += -L $(ERL_INTERFACE_LIB_DIR) -lerl_interface -lei
+LDLIBS += -L $(ERL_INTERFACE_LIB_DIR) -lei
 
 ifeq ($(UNAME_SYS), OpenBSD)
 	LDLIBS += -lestdc++

